content/blog/2022-05-08-Apache-ShenYu(incubating)-Integrated-Skywalking-Practice-Observability/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "SkyWalking ShenYu plugin implementation principles and observability practices"
```

Comment:
```suggestion
title: "Apache ShenYu(incubating) plugin implementation principles and observability practices"
```

Reply:
This is already in SkyWalking scope, so, don't have to mention SkyWalking anymore.